packages/react/src/components/Table/TableBody/_table-dnd.scss

```diff
@@ -0,0 +1,94 @@
+@import '../../../globals/vars';
+@import '../../../globals/spacing';
+
+// The 6-dots grable handle area on a row that can be dragged. This is really that the parent of
+// that picture so the draggable area is a larger, square.
+.#{$iot-prefix}--table-drag-handle {
+  height: 2rem;
+  padding-inline: 1rem;
+  display: grid;
+  align-items: center;
+  justify-items: center;
+
+  cursor: grab;
+  user-select: none;
+
+  position: relative;
+  inset-inline-start: var(--row-nesting-offset);
+}
+
+.bx--data-table td.#{$iot-prefix}--table-grab-handle-cell {
+  // remove padding so that the handle can fill the whole cell. Visually it's smaller, but the
+  // draggable region will be large.
+  padding-inline: 0;
+}
+
+// When hovering over a row. Draws a border around the row using an extra overlay element positioned
+// over it since CSS alone can add the require border around the row.
+.#{$iot-prefix}--table-drop-row-overlay {
+  // Abs positioned, but JavaScript will set its rect
+  position: absolute;
+  border: dashed 2px $carbon--blue-60;
+  pointer-events: none;
+  // This probably doesn't need to be very high, as long as it's higher than the table.
+  z-index: 1000;
```

Comment:
I don't know what you mean. Is there a sass var or something to look at? Or some example? I don't know about their z index manager.

Reply:
basically means that we cannot statically set a z-index. if a component requires a zindex we need pass that information as props so that something rendering that components can tell it the best zindex to use.  (ie, that value is going to change if we put this in a dialog, drawer, etc, and we need be able to tell it what the best zindex to use.